tools/wltests/agendas/sched-evaluation-small.yaml

```diff
@@ -59,6 +59,7 @@ workloads:
     workload_parameters:
       format: "mov_720p"
       duration: 3
+      landscape: True
```

Comment:
Is this needed for "example-exoplayer-simple.yaml" as well ?

Reply:
You right! I've got it fixed in the other PR but forgot to update this one as well... :-(

Reply:
Oh no... sorry, that's new... was not sure about that since it's "just" an example... but likely it's better to keep it consistent. I'll add to that too!

Reply:
Mmm... actually, looking at the description of that agenda:
```
The workload parameters (such as video playback duration) are left to the workload's default.
```
thus... we should probably not add anything. I guess the goal there is just to show a possible minimal agenda.